src/modules/commentCounter.test.js

```diff
@@ -0,0 +1,25 @@
+// tests/createLike.test.js
+
+// Import the mock function
+import commentCounter from './commentCounter.js';
+
+// Mocking the fetch function
+describe('Comments Counter', () => {
+  test('Count the number of comments', () => {
+    const myObject = [
+      {
+        0: 'Rabu',
+        1: 'Hussein',
+        2: 'Hashi',
+      },
+    ];
+    const newCount = commentCounter(myObject);
+    expect(newCount).toBe(1);
+  });
+
+  test('Count the number of comments', () => {
+    const myObject = [1, 2, 3];
+    const newCount = commentCounter(myObject);
+    expect(newCount).toBe(3);
+  });
+});
```

Comment:
- [ ]  Check the way you've configured Jest, cz right now I can not run the test successfully.  Please fix it and make sure that all test are passing and full fill the requirements. 
![image](https://github.com/husush23/capstone--2/assets/43877689/d517ef65-9517-40e9-9afa-7755a1917fe1)

- [ OPTIONAL] I think that you have a large number of packages in you packaje.json that you are not using.  please consider using the one you need in your project.